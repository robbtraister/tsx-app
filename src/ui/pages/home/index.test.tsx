import { render } from "@testing-library/react";

import { Home } from ".";

describe("home page", () => {
  it("simple", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
