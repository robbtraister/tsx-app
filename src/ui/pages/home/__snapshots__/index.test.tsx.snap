// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home page simple 1`] = `
<div>
  <h1>
    Home
  </h1>
</div>
`;
